rootfs/ingest/python/qserv/testdata/case01/Filter.json

```diff
@@ -1,9 +1,9 @@
 {
     "auth_key":"",
-    "database":"qservTest_case01_qserv",
+    "database":"test_case01_qserv",
```

Comment:
Haven't you migrated database names to?
```
qcase01
qcase02
...
```

Reply:
This is a test dataset for unit test, database name does not need to be the same than `qcase01`, which is used for integration tests.